src/compare-tree/key-path.js

```diff
@@ -0,0 +1,40 @@
+const KEY_DELIMETER  = '.'
+const PATH_DELIMETER = ','
```

Comment:
It is. Slipped my mind. I tried to keep this stuff pretty separate from the rest of the project, but this should be consolidated.